Rewrite FAQ.md and CONTRIBUTING.md (#38742)
iptv-org/iptv: FAQ.md

# Frequently Asked Questions

### My favorite channel is not on the playlist.

> Make sure the channel is not in the [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue) and see [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details. Some channels or content may not be provided in this repository due to copyright or GitHub terms of service issues.

Reviewer: ```suggestion
Make sure the channel is not [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue), see the [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details. Some channels or content may not be provided in this repository due to copyright or GitHub terms of service issues.
```

> Start by asking our community for help via [Discussions](https://github.com/orgs/iptv-org/discussions). It is quite possible that someone already has a link to the channel you need and they just haven't added it to our playlist yet.

Reviewer: ```suggestion
Start by asking our community for help via [Discussions](https://github.com/orgs/iptv-org/discussions). It is quite possible that someone already has a link to the channel you need and they just haven't added it to our playlists yet.
```

Keep in mind that not all TV channels have some publicly available source for viewing online, and in this case there is little we can do about it.

### My favorite channel has disappeared from the playlist. Would you add it back?

> Make sure the channel wasn't [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue) recently and see [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details.

Reviewer: ```suggestion
Make sure the channel wasn't [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue) recently and see the [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details.
```

The repository simply contains user-submitted links to publicly available video stream URLs, note that we have no control over the stream sources. Unfortunately, if there's no alternate source that have been intentionally made public there is little we can do about it.

### Why do I see a channel that is not intended for my country?

> See the Playlist Strucutre section of the [Contributing Guide.md](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#playlist-structure)

Reviewer: ```suggestion
See the Playlist Strucutre section of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#playlist-structure)
```

> If you believe there's still an issue with automated sorting you can request to edit the feed info in [Database](https://github.com/iptv-org/database/issues) 

Reviewer: ```suggestion
If you believe there's still an issue after automated sorting, you can request to edit the feed info in [Database](https://github.com/iptv-org/database/issues) 
```

Author: ```suggestion
If you still believe the broadcast area or language is wrong, you can request to edit the feed info in our [Database](https://github.com/iptv-org/database/issues)
```

Author: okay?

# Frequently Asked Questions

### My favorite channel is not on the playlist.

> Make sure the channel is not [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue), see the [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details. Some channels or content may not be provided in this repository due to copyright or GitHub terms of service issues.

Reviewer 2: ```suggestion
Make sure the channel is not [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue). Some channels or content may not be provided in this repository due to copyright or GitHub terms of service issues.
```

Same as https://github.com/iptv-org/iptv/pull/38742/files#r3369458358

> Start by asking our community for help via [Discussions](https://github.com/orgs/iptv-org/discussions). It is quite possible that someone already has a link to the channel you need and they just haven't added it to our playlists yet.

Reviewer 2: ```suggestion
Start by asking our community for help via [Discussions](https://github.com/iptv-org/iptv/discussions). It is quite possible that someone already has a link to the channel you need and they just haven't added it to our playlists yet.
```

It's probably best to redirect directly to the "Discussions" page within the **iptv** repository.

Keep in mind that not all TV channels have some publicly available source for viewing online, and in this case there is little we can do about it.

### My favorite channel has disappeared from the playlist. Would you add it back?

> Make sure the channel wasn't [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue) recently and see the [History of related issues](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for details.

Reviewer 2: ```suggestion
Make sure the channel wasn't [blocklisted](https://iptv-org.github.io/?q=is_blocked%3Atrue) recently.
```

Each blocked channel already includes a link to the original notification, so there is no need to search the CSV file.

<img width="391" height="104" alt="image" src="https://github.com/user-attachments/assets/1289a6cf-16f2-45ad-8721-8c9092c70a9b" />

Author: This is valid for channels that are already in the blocklist. As of now, History of related issues is the only place where you can check if a DMCA notice caused by content, by source or by restriction of sharing a channel

Reviewer 2: The messages on the website vary depending on the type of block.

<img width="389" height="111" alt="image" src="https://github.com/user-attachments/assets/fd64b59e-db94-4313-8a30-7b05e59d8f4f" />

Author: We had a history of blocking channels after DMCA notices because those also broadcast infrighting content or now belong to a copyright protecting group. This may happen in the middle of PR and might confuse some

Reviewer 2: Sorry, what might be confusing?

Reviewer 3: >We had a history of blocking channels after DMCA notices because those also broadcast infrighting content or now belong to a copyright protecting group. This may happen in the middle of PR and might confuse some

Is this about the channels that have not been added yet but there's a past DMCA by their owner?

Author: > Sorry, what might be confusing?

Sometimes it is not clear what global DMCAs we have that may affect the repository and PRs ahead of blocklist:
e.g. 
- Some more Warner Bros streams were banned after submerging with other companies
- We have added persian Telewebion into blocklist during PR verification because it was broadcasting the Champion League

Reviewer 2: First of all, if the channel isn't listed in the notification, it shouldn't be on the block list to begin with.

Reviewer 3: >First of all, if the channel isn't listed in the notification, it shouldn't be on the block list to begin with.

That makes sense. It's more like a preventive thing currently. 
Many Warner bros channels are in the blocklist without being in the original request. But it seems like in doing so we have avoided getting DMCA from companies in contract with Warner Bro in some countries.

The repository simply contains user-submitted links to publicly available video stream URLs, note that we have no control over the stream sources. Unfortunately, if there's no alternate source that has been intentionally made public there is little we can do about it.

### Why do I see a channel that is not intended for my country?

> See the Playlist Structure section of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#playlist-structure)

Reviewer 2: ```suggestion
See the "Playlist Structure" section of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#playlist-structure).
```

> If you still believe the broadcast area or language is incorrect, you can request to edit the feed info in our [Database](https://github.com/iptv-org/database/issues)

Reviewer 2: ```suggestion
If you still believe this is an error, you can submit a request to update the channel info in our [Database](https://github.com/iptv-org/database/issues). Once the request is approved, the link will be automatically moved to a different playlist.
```

### Are you planning to include a Video On Demand (VOD) to the playlist?

No.

### Why is a channel available on iptv-org.github.io but missing from the playlist?

The site is an extensive database of all existing TV channels in the world, and does not represent a list of streams in our possession. Only channels for which we have working stream links are included in the playlists.

> You may still check [iptv-org.github.io with the "streams:>0" parameter](https://iptv-org.github.io/?q=streams%3A%3E0) in the search bar to check all available streams.

Reviewer 2: ```suggestion
You may still check the site with the [streams:>0](https://iptv-org.github.io/?q=streams%3A%3E0) parameter in the search bar to see all available streams.
```

### Can I add a radio broadcast?

Yes, if it is a [visual radio](https://en.wikipedia.org/wiki/Visual_radio) in which a video and audio are shown at the same time.

### How do I find a new working link?

Find a publicly available stream using your favourite search engine or by searching GitHub.

> Carefully read Validation section of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Validation)

Reviewer 2: ```suggestion
Find a publicly available stream using your favourite search engine or by searching GitHub. Just don't forget to carefully read the "Validation" section in the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Validation) first.
```

### Why isn't my link accepted?

> See the Requirements and Validation sections of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Requirements)

Reviewer 2: ```suggestion
See the "Requirements" and "Validation" sections of the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Requirements).
```

### Why do you not accept links that work in my player of choice or in a browser?

> The goal of IPTV-ORG is to provide reliable playlists that can be used with a variety of players, on every platform and every capable device.

Reviewer 2: ```suggestion
The goal of **iptv-org** is to provide reliable playlists that can be used with a variety of players, on every platform and every capable device.
```

### Why is the channel on the iptv-org.github.io but not in the playlist?

The site is an extensive database of all existing TV channels in the world, and does not represent a list of streams in our possession. Only those of them for which we have working stream links are included in the playlists.
You may still check [iptv-org.github.io with the "streams:>0" parameter](https://iptv-org.github.io/?q=streams%3A%3E0) in the search bar to check all available streams.

### Can I add a radio broadcast?

Yes, if it is a [visual radio](https://en.wikipedia.org/wiki/Visual_radio) in which a video and audio are shown at the same time.

### How do I find a new working link?

Find publicly available stream using your favourite search engine or by searching across Github.
Carefully read Validation section of the [Contributing Guide.md](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Validation)

### Why isn't my link accepted?

See the Requirements and Validation sections of the [Contributing Guide.md](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#Requirements)

> ### Why do you not accept links that work in my player of choice or in a browser?

Author: As per https://github.com/orgs/iptv-org/discussions/561 I do not accept links that I can't open in VLC and my current guides explain why. However, I saw in testing files that we may accept KODI arguments. At this point none of the playlists have a source that need those arguments. Are we accepting KODI now? Do we have to verify if the link works in KODI as well?

Reviewer: It's quite niche for our usage right now as they're tons of other alternatives, but I hope not. Because if KODI support's official, welcome to DRM content and other kinds of protection bypass issues...
